### PR TITLE
Alekseybb197 bug #2785

### PR DESCRIPTION
**Pull request by alekseybb197**
# What does this change
Fix for Bug #2785 porter param list doesn't display params

Wrong line "cr, ok := v.(DisplayCredentialSet)" was changed to "cr, ok := v.(DisplayParameterSet)"

After it
❯ porter param list
---------------------------------------
  NAMESPACE  NAME          MODIFIED    
---------------------------------------
             porter-hello  2023-06-18

# What issue does it fix
Closes # bug #2785 

[1]: https://getporter.org/src/CONTRIBUTING.md#when-to-open-a-pull-request

# Notes for the reviewer

# Checklist
- [ ] Did you write tests?
- [ ] Did you write documentation?
- [ ] Did you change porter.yaml or a storage document record? Update the corresponding schema file.
- [ ] If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

# Reviewer Checklist
* Comment with /azp run test-porter-release if a magefile or build script was modified
* Comment with /azp run porter-integration if it's a non-trivial PR

[contributors]: https://getporter.org/src/CONTRIBUTORS.md

### CODE REVIEW

File: pkg/config/datastore.go [hunk 1]
```diff
@@ -37,6 +37,9 @@ type Data struct {
 	// By default, Porter requires the --force flag to be specified to overwrite a bundle or image.
 	ForceOverwrite bool `mapstructure:"force-overwrite"`
 
+	// Force skip rebuild invocation image even if out-of-date detected
+	AutoBuildDisabled bool `mapstructure:"autobuild-disabled"`
```
Comment (troy0820) [inline pkg/config/datastore.go:41]: Is this necessary for this PR?

**Review by troy0820 — APPROVED** [review]
This looks good, but the only question is about the `pkg/config/datastore.go`

**Review by troy0820 — CHANGES_REQUESTED** [review]
Could you squash these commits?

File: pkg/config/datastore.go (hunk 1 above)
Comment (alekseybb197) [reply pkg/config/datastore.go:41]: No. It came out in the process of resolving conflicts. Should I try to delete?

**Comment by schristoff** [comment]
Hey, it may be easier to just delete your local Porter repository, reclone, and then make a new PR with the necessary change  - this way we don't have to hone in our Git Fu but the Git Log is happy. Let me know what you think :)

**Comment by alekseybb197** [comment]
just doing it... )

**Comment by alekseybb197** [comment]
Hm...
https://github.com/getporter/porter/pull/2789/checks?check_run_id=14479827416
((
Ok. I will remove my fork and make a new one

**Pull request closed without merging**